modules/jupyter-widget/package.json

```diff
@@ -34,7 +34,7 @@
     "@deck.gl/json": "8.2.0-alpha.1",
     "@deck.gl/layers": "8.2.0-alpha.1",
     "@deck.gl/mesh-layers": "8.2.0-alpha.1",
-    "@jupyter-widgets/base": "^2",
+    "@jupyter-widgets/base": "^2 || ^3",
```

Comment:
This will never resolve to 2.x without a lock file. Either drop `^2` Or move it to `peerDependencies`.

Reply:
I thought it looked weird, but that's what was [suggested by a Jupyter developer](https://github.com/jupyter-widgets/ipywidgets/issues/2798#issuecomment-593716999).

I can remove `^2` and then test to make sure it still loads fine in Notebook and Lab 1.x and 2.x